src/main/java/com/github/davidcarboni/thetrain/json/Transaction.java

```diff
@@ -47,23 +57,101 @@
     public Map<String, List<String>> files;
 
 
+// TODO keep this until we know 100% what the issue is.
```

Comment:
remove?

Reply:
The was the original impl - I want to keep this until we know what the hell is going on.